R/subscript-tables.R

```diff
@@ -0,0 +1,38 @@
+#' @export
+`[.QTable` <- function(x, ..., drop) {
```

Comment:
Why not just default `drop` to `TRUE` to match base R?

Reply:
Updated in https://github.com/Displayr/verbs/pull/39/commits/719938ebb9c3390c79d2ae67b006f61694c80a8f